content/content-strategy/identify-business-needs/what-business-needs/strategic-pyramid-head.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
!! This file is not referenced in the `index.yml` file for the `/what-business-needs` topic. It will not appear on the page. Recommend: remove if not needed or add to the index.yml if you intend to keep it.